### PR TITLE
fix: border color change

### LINKED ISSUE #1502: [BUG] SfComponentSelect wrong border color
**Describe the bug**

**How to reproduce**

1. go to storybook
2. open other/form 
3. see different underline color on select than others inputs  

**Expected behavior**
SfComponentColor should have --c-link color border like other inputs.  

**Actual behavior**

**Code examples**
Sample code (or your configuration) to help us reproduce the bug faster or to understand your use case.
```
//Write code here
```
Or add code snippets (Code sandbox)

**Screenshots**
image

**Devices (please complete the following information):**
 - Device: [e.g. iPhone6, Desktop]
 - Browser: [e.g chrome, safari][version]

**Additional context**
Anything you would like to add but was not sure which section it fits. For example: tech stack used (Nuxt, Vue, Gridsome, React, etc)

### PR DESCRIPTION
**Pull request by AdamPawlinski**
# Related issue
Closes #1502 

# Scope of work
Change border color to --c-link. 

# Screenshots of visual changes
image

# Checklist

- [x] No commented blocks of code left
- [x] Run tests and docs
- [x] Self code-reviewed
- [ ] Changes documented 

If applicable:

- [ ] I followed [composition rules](https://docs.storefrontui.io/contributing/coding-guidelines.html#component-rules) for my component
- [ ] I tested the component in most common device sizes (can be tested in Storybook [from viewport addon in top menu](https://github.com/storybooks/storybook/tree/master/addons/viewport))

### CODE REVIEW

**Review by justyna-13 — COMMENTED** [review]
What do you think about changing also color of the label in SfSelect? Here we can see it's a bit different (darker) than other labels :)

**Comment by AdamPawlinski** [comment]
@justyna-13  I made changes you mentioned and also added v-if condition for placeholder in SfSelect to show it only when it's passed to a component. Unfortunately after merging all other fixes regarding select they all appeared here as they would be made in this fix.

File: packages/shared/styles/components/molecules/SfComponentSelect.scss [hunk 1]
```diff
@@ -1,189 +1,155 @@
...
-  &.is-active {
-    --component-select-option-background: var(--c-light);
-  }
-  @include for-desktop {
-    --component-select-option-padding: var(--spacer-xs);
-    &:hover {
-      --component-select-option-background: rgba(var(--c-gray-base), 0.1);
-    }
-  }
+    display: flex;
```
Comment (justyna-13) [inline packages/shared/styles/components/molecules/SfComponentSelect.scss:3]: would be good to have the same indentation as before :)

Comment (AdamPawlinski) [reply packages/shared/styles/components/molecules/SfComponentSelect.scss]: fixed, I think so...

**Review by justyna-13 — APPROVED** [review]
great! :1st_place_medal:

**Pull request merged**